static/js/analytics/support.js

```diff
@@ -15,8 +15,4 @@ $(() => {
             window.alert("The string_id you entered is not correct. Aborted.");
         }
     });
-
-    $("a.copy-button").on("click", function () {
```

Comment:
Why is this deleted?

Reply:
I was trying to reload the page using javascript. So i made the change in this file so while deleting that this got also deleted.. my bad :(